.cargo/config.toml

```
@@ -1,7 +1,7 @@
 # paths = ["/path/to/override"] # path dependency overrides
 
 [alias] # command aliases
-install_soroban = "install --git https://github.com/AhaLabs/soroban-tools --rev c7fb7e08ba8efa9828d9df863d991558f269e35b --root ./target soroban-cli --debug"
+install_stellar = "install --locked stellar-cli --version 21.0.0 --force --root ./target stellar-cli --debug"
```

We can probably pin to a major version with `^21` instead of pinning to a specific version.

I also don't think we need to set the root, or tell cargo to install with the dev profile with the debug flag.

```suggestion
install_stellar = "install --locked stellar-cli@^21 --force"
```

---

Ah ok I see the root is used to write the binary to a local directory then use it locally. Seems like an odd practice to encourage given it isn't encouraged in any other setup documentation though.